test/e2e/apimachinery/crd_publish_openapi.go

```diff
@@ -598,4 +598,4 @@ properties:
     properties:
       bars:
         description: List of Bars and their statuses.
-        type: array`)
+        type: object`)
```

Comment:
would it be better to define `items: {type: object}`?

Reply:
that's better. Updated